src/components/forms/ReviewNHSAppTemplate/ReviewNHSAppTemplate.tsx

```diff
@@ -47,6 +47,9 @@ export function ReviewNHSAppTemplate({
         }}
         PreviewComponent={<PreviewTemplate.NHSApp message={html} />}
       />
+      <p>
+        <a href={`${getBasePath()}/manage-templates`}>Back to all templates</a>
```

Comment:
For internal links that use default styling you can use the Next `<Link>` tag which will add the base path for you.

Reply:
<a> tags swapped for <Link> tags.